src/platform/mt8195/include/platform/drivers/interrupt.h

```diff
@@ -0,0 +1,152 @@
+#define L1_DSP_TIMER_IRQ0_B (INT_LEVEL(2) | INTERRUPT_ID(0))
+#define L1_DSP_TIMER_IRQ1_B (INT_LEVEL(3) | INTERRUPT_ID(1))
+#define L1_DSP_TIMER_IRQ2_B (INT_LEVEL(4) | INTERRUPT_ID(2))
+#define L1_DSP_TIMER_IRQ3_B (INT_LEVEL(5) | INTERRUPT_ID(3))
+
+#define LX_CQDMA_IRQ0_B (INT_LEVEL(1) | INTERRUPT_ID(4))
+#define LX_CQDMA_IRQ1_B (INT_LEVEL(1) | INTERRUPT_ID(5))
+#define LX_CQDMA_IRQ2_B (INT_LEVEL(1) | INTERRUPT_ID(6))
+#define LX_CQDMA_IRQ3_B (INT_LEVEL(1) | INTERRUPT_ID(7))
+#define LX_UART_IRQ_B (INT_LEVEL(1) | INTERRUPT_ID(8))
+#define LX_AFE_IRQ_B (INT_LEVEL(1) | INTERRUPT_ID(9))
+#define LX_MCU_IRQ_B (INT_LEVEL(1) | INTERRUPT_ID(10))
+#define LX_I2C_IRQ4_B (INT_LEVEL(1) | INTERRUPT_ID(11))
+#define LX_I2C_IRQ5_B (INT_LEVEL(1) | INTERRUPT_ID(12))
+#define LX_RSVD_IRQ1_B (INT_LEVEL(1) | INTERRUPT_ID(13))
+#define LX_RSVD_IRQ2_B (INT_LEVEL(1) | INTERRUPT_ID(14))
+#define LX_ASRC_IRQ0_B (INT_LEVEL(1) | INTERRUPT_ID(15))
+#define LX_ASRC_IRQ1_B (INT_LEVEL(1) | INTERRUPT_ID(16))
+#define LX_ASRC_IRQ2_B (INT_LEVEL(1) | INTERRUPT_ID(17))
+#define LX_ASRC_IRQ3_B (INT_LEVEL(1) | INTERRUPT_ID(18))
+#define LX_ASRC_IRQ4_B (INT_LEVEL(1) | INTERRUPT_ID(19))
+#define LX_ASRC_IRQ5_B (INT_LEVEL(1) | INTERRUPT_ID(20))
+#define LX_ASRC_IRQ6_B (INT_LEVEL(1) | INTERRUPT_ID(21))
+#define LX_ASRC_IRQ7_B (INT_LEVEL(1) | INTERRUPT_ID(22))
+#define LX_ASRC_IRQ8_B (INT_LEVEL(1) | INTERRUPT_ID(23))
+#define LX_ASRC_IRQ9_B (INT_LEVEL(1) | INTERRUPT_ID(24))
+#define LX_ASRC_IRQ10_B (INT_LEVEL(1) | INTERRUPT_ID(25))
+#define LX_ASRC_IRQ11_B (INT_LEVEL(1) | INTERRUPT_ID(26))
+#define LX_ASRC_IRQ15_IRQ12_B (INT_LEVEL(1) | INTERRUPT_ID(27))
+#define LX_SPM_IRQ_B (INT_LEVEL(1) | INTERRUPT_ID(28))
+#define LX_SCP_IRQ_B (INT_LEVEL(1) | INTERRUPT_ID(29))
+#define LX_MBOX_IRQ0_B (INT_LEVEL(23) | INTERRUPT_ID(32))
+#define LX_MBOX_IRQ1_B (INT_LEVEL(23) | INTERRUPT_ID(33))
+#define LX_MBOX_IRQ2_B (INT_LEVEL(23) | INTERRUPT_ID(34))
+#define LX_MBOX_IRQ3_B (INT_LEVEL(23) | INTERRUPT_ID(35))
+#define LX_MBOX_IRQ4_B (INT_LEVEL(23) | INTERRUPT_ID(36))
+#define LX_MISC_NNA0_IRQ0_B (INT_LEVEL(23) | INTERRUPT_ID(37))
+#define LX_MISC_NNA0_IRQ1_B (INT_LEVEL(23) | INTERRUPT_ID(38))
+#define LX_MISC_NNA0_IRQ2_B (INT_LEVEL(23) | INTERRUPT_ID(39))
+#define LX_MISC_NNA1_IRQ0_B (INT_LEVEL(23) | INTERRUPT_ID(40))
+#define LX_MISC_NNA1_IRQ1_B (INT_LEVEL(23) | INTERRUPT_ID(41))
+#define LX_MISC_NNA1_IRQ2_B (INT_LEVEL(23) | INTERRUPT_ID(42))
+#define LX_ADSP_TIMTER_IRQ0_B (INT_LEVEL(23) | INTERRUPT_ID(43))
+#define LX_ADSP_TIMTER_IRQ1_B (INT_LEVEL(23) | INTERRUPT_ID(44))
+#define LX_ADSP_TIMTER_IRQ2_B (INT_LEVEL(23) | INTERRUPT_ID(45))
+#define LX_ADSP_TIMTER_IRQ3_B (INT_LEVEL(23) | INTERRUPT_ID(46))
+#define LX_OS_TIMTER_IRQ_B (INT_LEVEL(23) | INTERRUPT_ID(47))
+
+#define MAX_IRQ_NUM 64
+#define DOMAIN1_MAX_IRQ_NUM 32
+#define MAX_INT_LEVEL 24
+#define AUDIO_IRQn LX_AFE_IRQ_B
+/*
+ *      irq register information
+ *      INT 23/24
+ *
+ */
+#define DSP_IRQ_POL (MTK_DSP_RG_DSP_IRQ_POL)
```

Comment:
No need for parentheses here - like https://github.com/thesofproject/sof/pull/4679/commits/b861af40bedbe5cdb9469f10e151097a4bcddfcd#diff-29b66f8b8cb291815b9816efca83bd849b1f3f04cd7710eb1d09f1846ba1c072R32